src/vivarium_testing_utils/automated_validation/data_transformation/formatting.py

```diff
@@ -40,11 +40,33 @@ class TransitionCounts(SimDataFormatter):
     """Formatter for simulation data that contains transition counts."""
 
     def __init__(self, cause: str, start_state: str, end_state: str) -> None:
-        super().__init__("transition_count", cause, f"{start_state}_to_{end_state}")
+        super().__init__(
+            measure="transition_count",
+            entity=cause,
+            filter_value=f"{start_state}_to_{end_state}",
+        )
 
 
 class PersonTime(SimDataFormatter):
     """Formatter for simulation data that contains person time."""
 
     def __init__(self, cause: str, state: str | None = None) -> None:
-        super().__init__("person_time", cause, state or "total")
+        super().__init__(
+            measure="person_time",
+            entity=cause,
+            filter_value=state or "total",
+        )
+
+
+class TotalPersonTime(SimDataFormatter):
```

Comment:
Perhaps both this and the `StatePersonTime` class could inherit from a more generic `PersonTime` class?